pkg/validationfile/loader_test.go

```diff
@@ -35,6 +35,18 @@ func TestPopulateFromFiles(t *testing.T) {
 				tuple.Parse("example/project:pied_piper#writer@example/user:freyja"),
 			},
 		},
+		{
+			name:      "multiple files",
+			filePaths: []string{"testdata/initial_schema_and_rels.yaml", "testdata/just_rels.yaml"},
```

Comment:
what happens if the first file doesn't have a schema, what happens if there are conflicting schemas?

Reply:
It'll raise an error saying the types aren't defined

Reply:
As discussed, it isn't raising an error at all, which I'll address in an immediate followup PR